components/node/cmd/batch_decoder/main.go

```diff
@@ -89,17 +89,17 @@ func main() {
 			Name:  "reassemble",
 			Usage: "Reassembles channels from fetched batches",
 			Flags: []cli.Flag{
-				cli.StringFlag{
+				&cli.StringFlag{
```

Comment:
Please change the default value to `0xff00000000000000000000000000000000000255`.